lib/stateView/document/mongoDbRepository/SVDocumentMongoDbRepository.js

```diff
@@ -93,13 +93,15 @@ class SVDocumentMongoDbRepository {
    * @param [query.startAt]
    * @param [query.startAfter]
    * @param [query.orderBy]
+   * @param {Object} [documentSchema]
    * @param {MongoDBTransaction} [stateViewTransaction]
    *
    * @returns {Promise<SVDocument[]>}
    * @throws {InvalidQueryError}
    */
-  async fetch(query = {}, stateViewTransaction = undefined) {
-    const result = this.validateQuery(query);
+
+  async fetch(query = {}, documentSchema = {}, stateViewTransaction = undefined) {
```

Comment:
Why document schema as optional?

Reply:
Because query can be empty. And we don't need it, if it is empty